docs/resources/quality_monitor_v2.md

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Quality Monitor"
+---
+# databricks_quality_monitor_v2 Resource
+Users with MANAGE Schema can use quality monitor v2 to set up anomaly detection checks for UC objects, currently support schema. 
+
+
+-> **Note** This resource can only be used with an workspace-level provider!
+
+
+## Example Usage
+```hcl
+resource "databricks_quality_monitor_v2" "this" {
+  object_type = "schema"
+  object_id = "ecb4f03c-f6c9-4d84-8934-e50e087d2435
```

Comment:
it's not very user-friendly API imho...

Reply:
Hi @alexott , is the concern regarding object_id specified to some exact value? the examples are added by the team so if any changes are required here then it might be good to start a thread with the team.